tornado-api/src/main/java/uk/ac/manchester/tornado/api/ImmutableTaskGraph.java

```diff
@@ -170,10 +170,6 @@ void enableProfiler(ProfilerMode profilerMode) {
         taskGraph.enableProfiler(profilerMode);
     }
 
-    void disableProfiler(ProfilerMode profilerMode) {
```

Comment:
I see that we also do not use the `enableProfiler` method. I guess it is because we have it in the `TornadoExecutionPlan`. 

In any case, do you remove the method because it is unused?

Reply:
The intenal logic has been refactored. We can use the `ExecutionFrame` to pass the profiler mode down to the `TornadoTaskGraph`.

Reply:
in this case, shall we also remove the `enableProfiler` method?